### DISCUSSION TITLE
HARMAN SUPPORT

### DISCUSSION THREAD

**Discussion by JonWhitefield** [airsdk/Adobe-Runtime-Support#624] [Q&A]
Hi All, 

First, apologies this is not a bug but please hear my plight. I have been trying to contact Harman since the middle of June and cannot get a response.  I was told by a user of Adobe Air Forums that there were Harman people over here. 

I originally sent this to Adobe.Support@harman.com in the middle June and subsequently posted in the Adobe Forums and am yet to get a response. I am just looking for some clarification in my understanding regarding the transition happening at the end of the year.

This is how I currently understand it

- There will be no Harman Air 33 runtime software akin to the current Air 32 runtime. 
- As a result, all applications will have to be built with captive runtimes
- Developers in the short term will have to bundle their applications into native installers using 3rd party software
- The application update service will be lost due to the lack of air installer, see below

![image](https://user-images.githubusercontent.com/72615768/95596660-998b5a80-0a45-11eb-9073-33e3376e22ba.png)

Currently we do not distribute our application as a captive runtime and we make use of the adobe installer and update services. I just want to ensure that if we have to pivot to support the new way of distributing our applications that we will not need to re-think it again in 6-12 months.

 

Thanks for any thoughts or input.  Sorry for placing this into a bug tracker
Jon

**Comment by ajwfrost** [comment]
Hi

Apologies at the lack of response, the above message looks familiar so I think I've seen it but thought I'd replied; the Adobe.Support inbox did get a little behind although we should have more people going through things now..

In term of the query: we're actually planning to have a 'shared runtime' for AIR 33, to replace this from Adobe, although we would still prefer people to use the captive runtime approach if that's feasible. We are looking to update ADT so that the 'native' packaging can also include the captive runtime, to make this easier.

In terms of updating .. the automatic updating of the runtime itself I think is already turned off in AIR 32, but this wouldn't have been possible anyway as Adobe's last release was out and we don't yet have an online service for automatic AIR runtime updates. This is something we may look into adding in the future although I don't know how many people would want this. So for now for the shared AIR runtime, it would need to be manual installs/updates from our website.

For the application update service, we will need to look into this further, although I would hope that this still works in the same way.

Feel free to email me at Andrew.Frost at Harman.com and we can look at getting you a beta version of the shared runtime to ensure that you're confident with this approach.


thanks

   Andrew

**Comment by JonWhitefield** [comment]
Thanks for your responce Andrew its very much appreciated. Thats has cleared up a few of my points, If I can send you a followup  email in the not too distant future that would be great. 

Thanks,
Jon

**Comment by navbhandari1990** [comment]
Can you please guide me how do I get harman's package browser solution for my organization to use flash for internal application post 2020 ? Sending email to harman support  is not helping as no response. Please let me know what is the process

**Comment by ajwfrost** [comment]
@navbhandari1990 that email address should be being monitored by one of the marketing teams for this sort of enquiry.. let me check and I’ll try to get someone to contact you.
Thanks
